src/actions/fetchInstitutions.js

```diff
@@ -3,7 +3,7 @@ import receiveInstitutions from './receiveInstitutions.js'
 import receiveError from './receiveError.js'
 import hasHttpError from './hasHttpError.js'
 import requestInstitutions from './requestInstitutions.js'
-import { getInstitutions } from '../api/api.js'
+//import { getInstitutions } from '../api/api.js'
```

Comment:
This action is no longer used at all. We can remove it but wanted to let it for now. Let me know what you think.